pkg/controller/daemon/controller.go

```diff
@@ -207,7 +203,7 @@ func NewDaemonSetsController(podInformer framework.SharedIndexInformer, kubeClie
 }
 
 func NewDaemonSetsControllerFromClient(kubeClient clientset.Interface, resyncPeriod controller.ResyncPeriodFunc, lookupCacheSize int) *DaemonSetsController {
-	podInformer := informers.CreateSharedPodIndexInformer(kubeClient, resyncPeriod())
+	podInformer := informers.CreateSharedPodIndexInformer(kubeClient, controller.NoResyncPeriodFunc())
```

Comment:
Sorry for delay. So I'm torn here. A quick rescan doesn't show anything that is obviously broken without resyncs however I am worried about bugs. I would have the same concern with other controllers so if we have decided to do this with other controllers I guess we should do this with daemonsetcontroller as well. Also we are relatively early in the v1.4 merge window so this is a good time to do it.